package.json

```diff
@@ -36,7 +36,7 @@
     "postinstall": "husky install"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,json,sol}": [
+    "*.{js,jsx,ts,tsx,json}": [
```

Comment:
Couldn't this PR be rebased on a more current commit from `development` so that I don't need to see redundant changes from all the other PRs I reviewed?